account_payment_credit_card/readme/USAGE.rst

```diff
@@ -0,0 +1,10 @@
+
+To use this module, you need to:
+
+#. Go to Accounting > Purchases > Vendors Bills
+#. Select an open bill and click on Register Payment
+#. Select the credit card journal and validate the payment
+
+In Journals, Selected the Transfer AP to Credit Card Company & Credit Card Company Name(Partner), Then In Invoice move to Register Payment and Done Payment.
+In Account move line Added a new Lines with Credit Card Company Name(Partner) & ref and other fields.
+
```

Comment:
Please remove empty line

Reply:
@nikul-serpentcs this one is still there

Reply:
> @nikul-serpentcs this one is still there

It's the Last line for this file, that's why not removed
If I removed then throw Error `add a new blank line`

Reply:
Odd, this one doesn't have an empty line in the end - https://github.com/OCA/maintainer-tools/blame/master/template/module/readme/USAGE.rst - only newline. If possible, please provide more details on the error - which tool gives it

Reply:
> Odd, this one doesn't have an empty line in the end - https://github.com/OCA/maintainer-tools/blame/master/template/module/readme/USAGE.rst - only newline. If possible, please provide more details on the error - which tool gives it

In .py and .xml file get Error, So I added a blank line for that
anyway, I removed It, review it!